From 33271cc2b60d56cd0272f9126f5230e21d8afa97 Mon Sep 17 00:00:00 2001
From: Etienne Brateau <etienne.brateau@gmail.com>
Date: Tue, 18 Mar 2025 23:55:35 +0100
Subject: [PATCH 2/2] Fix the extension of the shared library for GNU/Hurd

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 2ce361e..bb8d440 100755
--- a/configure
+++ b/configure
@@ -724,7 +724,7 @@ elif darwin; then
   SHARED_EXT=".${RHASH_VERSION}.dylib"
   SO_MAJ_EXT=".${RHASH_VERSION_MAJOR}.dylib"
   SOLINK_EXT=".dylib"
-elif linux; then
+elif linux || gnu; then
   # use the full library version for the library file extension
   SHARED_EXT=".so.${RHASH_VERSION}"
 fi
-- 
2.49.0

