From ac5a24c82f64a0a75ac54861818de5a885f0d605 Mon Sep 17 00:00:00 2001
From: Etienne Brateau <etienne.brateau@gmail.com>
Date: Sun, 6 Jul 2025 13:30:07 +0200
Subject: [PATCH] Fix build of idmapping.c on hurd

---
 lib/idmapping.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/idmapping.c b/lib/idmapping.c
index ef685f32..f4729f14 100644
--- a/lib/idmapping.c
+++ b/lib/idmapping.c
@@ -12,8 +12,8 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <strings.h>
-#include <sys/prctl.h>
 #if __has_include(<sys/capability.h>)
+#include <sys/prctl.h>
 # include <sys/capability.h>
 #endif
 
-- 
2.50.0

